tests/views.py

```diff
@@ -0,0 +1,2 @@
+def test_failed_response(request, error_message, status):
+    pass
```

Comment:
Empty test?

Reply:
No, it's a view function to ensure that the real resolution of a dotted path works.  The test that uses this is here:

https://github.com/atdsaa/django-auth-adfs/blob/custom_error_handler/tests/test_settings.py#L50

I didn't want to use the lambda because I wanted to ensure there was no issues with loading the function during app start up. One of first attempts resulted in a circular reference.

Reply:
Ah, I see. Thank you.